index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="kr">
```

Comment:
```suggestion
<html lang="ko">
```

HTMLElement.lang속성은 요소의 특성값과 텍스트 내용의 기본언어를 가져오거나 설정합니다! 이때 보통은 '언어'로 설정이 되고, 특별히 지역특성을 강조하고자 할 때는 '언어-지역(대문자)' 로 설정을 한답니다!

[MDN](https://developer.mozilla.org/ko/docs/Web/HTML/Global_attributes/lang)과 [이부분](https://codingeverybody.kr/html-lang-%EC%86%8D%EC%84%B1-%EC%98%AC%EB%B0%94%EB%A5%B8-%EC%9D%B4%ED%95%B4%EC%99%80-%EC%82%AC%EC%9A%A9-%EB%B0%A9%EB%B2%95/)참고부탁드릴게요!